examples/rosettacode/read_entire_file.nit

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: Read entire file
+#
+# SEE: http://rosettacode.org/wiki/Read_entire_file
+module read_entire_file
+
+var filename = "/etc/issue"
+var content  = filename.to_path.read_all
```

Comment:
because the task was about getting the content, not doing something with.
So to explicitly solve the task I explicitly identify the content.